packages/@aws-cdk-testing/framework-integ/test/aws-logs/test/integ.log-retention.ts

```diff
@@ -1,21 +1,27 @@
-import { App, Stack, StackProps, RemovalPolicy } from 'aws-cdk-lib';
+import { App, Stack, StackProps, RemovalPolicy, Tags } from 'aws-cdk-lib';
 import { IntegTest } from '@aws-cdk/integ-tests-alpha';
 import { LogRetention, RetentionDays } from 'aws-cdk-lib/aws-logs';
 
 class LogRetentionIntegStack extends Stack {
   constructor(scope: App, id: string, props?: StackProps) {
     super(scope, id, props);
 
-    new LogRetention(this, 'MyLambda', {
-      logGroupName: 'logRetentionLogGroup',
+    Tags.of(this).add('env', 'prod');
```

Comment:
We should add assertions for both integ tests to assert that the tags have been
added.

Reply:
I've added assertions to both log-retention integ tests.